Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -89,6 +84,19 @@ final class SettingsViewController: UIViewController {
       .observeValues { [weak self] link in self?.goToAppStore(link: link) }
   }
 
+  private func leftBarButtonItem() -> UIBarButtonItem {
+    let leftBarButtonItem = UIBarButtonItem(
+      image: UIImage(named: "icon--cross"),
+      style: .plain,
+      target: self,
+      action: #selector(closeButtonPressed)
+    )
+    leftBarButtonItem.accessibilityLabel = Strings.Dismiss()
```

Comment:
Is it possible to use the key path setters here?
`_ = leftBarButtonItem |> \.accessibilityLabel %~ { _ in Strings.Dismiss }`